src/Compilers/CSharp/Portable/Compilation/InitializerSemanticModel.cs

```diff
@@ -90,24 +90,23 @@ internal override BoundNode GetBoundRoot()
             switch (rootSyntax.Kind())
             {
                 case SyntaxKind.VariableDeclarator:
-                    rootSyntax = ((VariableDeclaratorSyntax)rootSyntax).Initializer.Value;
+                    rootSyntax = ((VariableDeclaratorSyntax)rootSyntax).Initializer;
```

Comment:
`GetBoundRoot` is only used by the region analysis APIs. This change ensures that we have a bound node for the equals value, which ensures that the SemanticModel will generate the correct the correct `ISymbolInitializerOperation` nodes for the current context, and the extra node does not impact the results of the region analysis apis.